doc/api/next_api_changes/deprecations/23455-OG.rst

```diff
@@ -0,0 +1,5 @@
+``w_xaxis``, ``w_yaxis``, and ``w_zaxis`` of `~.Axis3D`
```

Comment:
Avoid links in the title.

Reply:
Especially those that doesn't work... This was fixed to sort out the failing build (and I also read a bit about it by coincidence...).